src/app/table/table.component.ts

```diff
@@ -129,7 +139,17 @@ export class GpfTableComponent implements OnChanges, AfterViewChecked {
   }
 
   public getVisibleData(): Array<any> {
-    this.tableWidth = this.tableViewChild?.nativeElement.offsetWidth;
+    this.tableWidth = '0';
+    if (this.columnsChildren !== undefined && this.columnsChildren !== null) {
+      this.columnsChildren.forEach(column => {
+        this.tableWidth = String(parseFloat(this.tableWidth) + parseFloat(column.columnWidth));
+      });
+      this.tableWidth += 'px';
```

Comment:
Don't convert this to string and float on every loop, just work with a float value and convert it to string after the loop has finished.